### DISCUSSION TITLE
LCP increase after >=13.5.5-canary.1

### DISCUSSION THREAD

**Discussion by teh23** [vercel/next.js#57989] [App Router]
13.5.5-canary.0:
![image](https://github.com/vercel/next.js/assets/14942614/d9ace548-5839-4589-a08c-d8d0a47056ac)

13.5.5-canary.1:
![image](https://github.com/vercel/next.js/assets/14942614/ea420f72-88df-4541-b6ac-93582a657858)

14.0.1:
![image](https://github.com/vercel/next.js/assets/14942614/545083ca-e808-42e3-8c3b-c73b77db5e3e)

Performance tab:

13.5.5-canary.0:
![image](https://github.com/vercel/next.js/assets/14942614/8d87f16a-ff99-431c-87f4-685e3bb440a1)

14.0.1:
![image](https://github.com/vercel/next.js/assets/14942614/b336f902-7be8-4744-9406-bdc074773b09)

Does anyone have any idea what could be the cause of such a spike? Or maybe someone else had a similar issue?

**Comment by icyJoseph** [comment]
The lighthouse runs have a breakdown of the 4 parts involved in LCP:

Something like:

LCP sub-part | % of LCP
-- | --
Time to first byte (TTFB) | ~40%
Resource load delay | <10%
Resource load time | ~40%
Element render delay | <10%
TOTAL | 100%

Could you posts yours?

**Comment by teh23** [comment]
Sure!

13.5.5-canary.0:

Phase | % of LCP | Timing
-- | -- | --
TTFB | 14% | 450 ms
Load Delay | 0% | 0 ms
Load Time | 0% | 0 ms
Render Delay | 86% | 2,720 ms

14.0.1:

Phase | % of LCP | Timing
-- | -- | --
TTFB | 10% | 450 ms
Load Delay | 0% | 0 ms
Load Time | 0% | 0 ms
Render Delay | 90% | 4,100 ms

**Comment by markoleavy** [comment]
@teh23 have you found any solution? I also have really long render delay. As mention on discussion [58982](https://github.com/vercel/next.js/discussions/58982) it might be because of the async loading of next javascripts.
 PHASE | % of LCP | Timing
-- | -- |--
TTFB | 12% | 630 ms
Load Delay | 11% | 620 ms
Load Time | 2% | 120 ms
Render Delay | 75% | 4.020 ms

**Comment by teh23** [comment]
@markoleavy We haven't found a solution yet, we are still on 13.5.4 with this project. Also we had another project and we were facing the same issue with high render delay. Both sites are static and have a lot of data loading from rsc into client component maybe it is something in this direction.

I tested this async script by editing .next/server/app/index.html no effect at all

**Reply by markoleavy** [reply]
Did you switched all of the next scripts to defer?

**Reply by markoleavy** [reply]
Just tried to switch back to 13.5.4 and my LCP moved from 5-6 seconds to 2.7.
I can say there's a huge difference in the number of scripts in the doc's head between the two version:
Next-related scripts on 13.5.4:
```jsx
<script src="/_next/static/chunks/afbc845c-5d36f2573a2b591d.js" async="" crossorigin=""></script>
<script src="/_next/static/chunks/6231-4e14ec391dfe872e.js" async="" crossorigin=""></script>
<script src="/_next/static/chunks/main-app-2635d5f757a6ff44.js" async="" crossorigin=""></script>Ï
```
On 14.1.0:
```jsx
<script src="/_next/static/chunks/a130e3cb-d38efdea30cfec3b.js" async="" crossorigin=""></script>
<script src="/_next/static/chunks/4237-681d4002027859ea.js" async="" crossorigin=""></script>
<script src="/_next/static/chunks/main-app-c2a7c0b582017c36.js" async="" crossorigin=""></script>
<script src="/_next/static/chunks/4027-1c8e6416e910c85c.js" async=""></script>
<script src="/_next/static/chunks/827-dfe3f0537e505d05.js" async=""></script>
<script src="/_next/static/chunks/491-eb8a89f37ca70982.js" async=""></script>
<script src="/_next/static/chunks/3994-d0c60c9a80fbbe35.js" async=""></script>
<script src="/_next/static/chunks/1666-fb1ccf53037735e0.js" async=""></script>
<script src="/_next/static/chunks/app/layout-2f513177a25f9f4b.js" async=""></script>
<script src="/_next/static/chunks/7441-4a9201d8e11007e6.js" async=""></script>
<script src="/_next/static/chunks/4701-9a49f4b78f98791b.js" async=""></script>
<script src="/_next/static/chunks/6425-5f8926146141af22.js" async=""></script>
<script src="/_next/static/chunks/3302-ce60431bc4a5e281.js" async=""></script>
<script src="/_next/static/chunks/880-2311daad7a04e2f7.js" async=""></script>
<script src="/_next/static/chunks/59-163a4eeb14d23ebb.js" async=""></script>
<script src="/_next/static/chunks/6844-fdfec061e872d372.js" async=""></script>
<script src="/_next/static/chunks/2678-aa6ed48349311666.js" async=""></script>
<script src="/_next/static/chunks/4658-8fe48b577eae9c76.js" async=""></script>
<script src="/_next/static/chunks/5164-7d5b0ec5527c32ad.js" async=""></script>
<script src="/_next/static/chunks/8901-4134a0830ec14388.js" async=""></script>
<script src="/_next/static/chunks/8459-6467e41a2881881f.js" async=""></script>
<script src="/_next/static/chunks/177-e5113f7b1a8f45d4.js" async=""></script>
<script src="/_next/static/chunks/2515-c3ae39a4c6ef89f7.js" async=""></script>
<script src="/_next/static/chunks/5573-96e397d8f7a1ec69.js" async=""></script>
<script src="/_next/static/chunks/2657-2bbf3058b7ef121a.js" async=""></script>
<script src="/_next/static/chunks/2070-7dc1e116ca2fa6ca.js" async=""></script>
<script src="/_next/static/chunks/3907-09b53e2850808f41.js" async=""></script>
<script src="/_next/static/chunks/4330-5f52fc3593fe75a0.js" async=""></script>
<script src="/_next/static/chunks/app/(mainLayout)/%5B...slug%5D/page-01d01e70c91c2bbf.js" async=""></script>
<script src="/_next/static/chunks/1135-31a778a9d5e21b45.js" async=""></script>
<script src="/_next/static/chunks/app/(mainLayout)/layout-862838f48f69d244.js" async=""></script>
<script src="/_next/static/chunks/app/(mainLayout)/loading-1058ffc73dc90d15.js" async=""></script>
```

**Comment by JacobBSargent** [comment]
I'm dealing with a similar issue, up to 11 seconds! With ``"next": "^15.0.3",``

Has there been any update here?


Phase | % of LCP | Timing
-- | -- | --
TTFB | 5% | 600 ms
Load Delay | 0% | 0 ms
Load Time | 0% | 0 ms
Render Delay | 95% | 11,730 ms
